src/pages/registration/resource_type_tab/SelectRegistrationTypeField.tsx

```diff
@@ -220,79 +228,156 @@ export const SelectRegistrationTypeField = () => {
             </IconButton>
           )}
         </Box>
-        <Box sx={{ display: 'flex', gap: '0.5rem', alignItems: 'center', justifyContent: 'end', mb: '1rem' }}>
-          <FilterVintageIcon
-            color="primary"
-            titleAccess={t('registration.resource_type.nvi.can_give_publication_points')}
-            fontSize="small"
+        <Box
+          sx={{
+            display: 'flex',
+            justifyContent: 'space-between',
+            flexWrap: 'wrap',
+            padding: '0.5rem',
+            gap: '0.5rem',
+            my: '1rem',
+          }}>
+          <TextField
+            data-testid={dataTestId.registrationWizard.resourceType.resourceTypeSearchField}
+            sx={{ maxWidth: '17rem' }}
+            type="search"
+            variant="outlined"
+            label={t('common.search')}
+            InputProps={{
+              startAdornment: <SearchIcon />,
```

Comment:
Syns fortsatt denne er litt finere med ikon til slutt, og filled altså, sånn som resten av feltene i wizarden. Får den hvite bakgrunnen, i tillegg til at label vil fungere som placeholder før den får fokus
```suggestion
            variant="filled"
            label={t('common.search')}
            InputProps={{
              endAdornment: <SearchIcon />,
```